pages/specification/migration/_index.md

```diff
@@ -5,76 +5,76 @@ section: docs
+| `exclusiveMaximum`      | Validation    | 03                 | No      | Replaced `maximumCanEqual`     |
+| `exclusiveMinimum`      | Validation    | 03                 | No      | Replaced `minimumCanEqual`     |
+| `extends`               | Validation    | 03                 | Yes     | No                             |
+| `format`                | Validation    | 02                 | No      | -                              |
+| `id`                    | Core          | 04                 | No      | Replaced by `$id`              |
+| `if`                    | Core          | 07                 | No      | No                             |
+| `items`                 | Core          | 2019-09            | No      | Replaced by `prefixItems`      |
+| `maxContains`           | Validation    | 2019-09            | No      | No                             |
+| `maxProperties`         | Validation    | 04                 | No      | No                             |
+| `maximumCanEqual`       | Validation    | 02                 | No      | Replaced by `exclusiveMaximum` |
+| `minimumCanEqual`       | Validation    | 02                 | No      | Replaced by `exclusiveMinimum` |
+| `minContains`           | Validation    | 2019-09            | No      | No                             |
+| `minProperties`         | Validation    | 04                 | No      | No                             |
+| `multipleOf`            | Validation    | 04                 | No      | Replaced `divisibleBy`         |
+| `not`                   | Core          | 04                 | No      | No                             |
+| `oneOf`                 | Core          | 04                 | No      | No                             |
+| `optional`              | Validation    | 02                 | No      | Replaced by `required`         |
+| `pattern`               | Core          | 03                 | No      | No                             |
+| `patternProperties`     | Core          | 03                 | No      | No                             |
+| `prefixItems`           | Core          | 2020-12            | No      | Replaced `items`               |
+| `propertyNames`         | Core          | 06                 | No      | No                             |
+| `readOnly`              | Validation    | 07                 | No      | No                             |
+| `required`              | Validation    | 03                 | No      | No                             |
+| `requires`              | Validation    | 02                 | Yes     | -                              |
+| `title`                 | Validation    | 01                 | No      | No                             |
+| `then`                  | Core          | 07                 | No      | No                             |
+| `type`                  | Validation    | 02                 | No      | No                             |
```

Comment:
`type` had a significant change in draft-04. It used to allow schemas as values and "any".

```suggestion
| `type` (legacy)         | Core          | 01                 | 04      | Replaced by `type`             |
| `type`                  | Validation    | 04                 | No      | Replaced `type` (legacy)       |
```